src/components/sdk/introduction.stories.mdx

```diff
@@ -6,15 +6,23 @@ SDK Components provide you the capability to build out a realtime meeting experi
 
 SDK components are aimed at accelerating your Amazon Chime SDK application development.
 
+For the majority of use cases, a chime application consists of **a server application** and **a client application**.
+
+* A server application – Manages meeting and attendee resources, and serves those resources to the client application. The server application is created in the AWS account and must have access to the IAM role mentioned previously.
+* A client application – Receives meeting and attendee information from the server application, and uses that information to make media.
```

Comment:
Why do we have two of these intro stories?

Reply:
I'm afraid that user may skip some of the section in the storybook. But you are right, I only keep the one in the [home page](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/introduction--page)